### PR TITLE
changing maintainer

### PR DESCRIPTION
**Pull request by tjtnew**
as per https://github.com/reconhub/incidence2/issues/30#issuecomment-667145989

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -4,9 +4,11 @@ Title: Compute, Handle, Plot and Model Incidence of Dated Events
 Version: 1.7.2
 Authors@R: c(
   person("Thibaut", "Jombart", role = c("aut"), email = "thibautjombart@gmail.com"), 
-  person("Zhian N.", "Kamvar", role = c("aut", "cre"), email = "zkamvar@gmail.com",
+  person("Zhian N.", "Kamvar", role = "aut", email = "zkamvar@gmail.com",
          comment = c(ORCID = "0000-0003-1458-7108")),
-  person("Rich", "FitzJohn", role = "aut", email = "rich.fitzjohn@gmail.com"), 
+  person("Rich", "FitzJohn", role = "aut", email = "rich.fitzjohn@gmail.com"),
+  person("Tim", "Taylor", role = "cre", email = "tim.taylor@hiddenelephants.co.uk",
```
Comment (zkamvar) [inline DESCRIPTION:10]: Is there a reason why you chose this place in particular over just adding your name at the end?

**Review by zkamvar — APPROVED** [review]
I just sent the email. I have one minor question, but you can ignore it if you want.

File: DESCRIPTION (hunk 1 above)
Comment (tjtnew) [reply DESCRIPTION:10]: No. why?

Comment (zkamvar) [reply DESCRIPTION:10]: Just curious about the choice. Academics tend to get fidgety about name order and a random placement is, well, random. That being said, this is your project now, so you're free to arrange the authors in any method you wish.

Comment (thibautjombart) [reply DESCRIPTION:10]: Thanfully the battles on name ordering are still usually confined to peer reviewed publications - haven't seen much arguments on `DESCRIPTION` files yet ¯\_(ツ)_/¯

**Pull request merged**